tests/support/globalSetup.ts

```diff
@@ -1,31 +1,74 @@
 import type { FullConfig } from "@playwright/test";
 
-import { seedPort, seedRemote } from "@tests/support/fixtures.js";
+import * as Fs from "node:fs/promises";
+import * as Path from "node:path";
+import * as readline from "node:readline/promises";
+import { createSeedFixture, tmp } from "./fixtures.js";
+import { execa } from "execa";
 
 export default async function globalSetup(_config: FullConfig): Promise<void> {
-  await assertHttpApiRunning();
+  try {
+    await assertRadInstalled();
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
+
+  console.log("Setting up global test environment");
+  if (!process.env.CI) {
+    await promptWorkspaceRemoval();
+  }
+  await removeWorkspace();
+  await createSeedFixture();
+}
+
+// Assert that the `rad` CLI is installed and has the correct version.
+async function assertRadInstalled(): Promise<void> {
+  const versionConstraint = (
+    await Fs.readFile(`${process.cwd()}/heartwood-version`, "utf8")
+  ).substring(0, 7);
+  const { stdout: version } = await execa("rad", ["--version"]);
+  if (!version.includes(versionConstraint)) {
+    throw new Error(
+      `rad version ${version} does not satisfy ${versionConstraint}`,
+    );
+  }
 }
 
-// Assert that the test http-api is running. If it is not running, throw an
-// error that explains how to run it.
-async function assertHttpApiRunning(): Promise<void> {
-  const notRunningMessage =
-    "The http-api server with test fixtures needs to be running.\n" +
-    "👉 You can start it with `./scripts/run-httpd-with-fixtures`\n";
+async function promptWorkspaceRemoval(): Promise<void> {
+  console.log("");
+  console.log("This will irrevocably destroy the following directories:");
+  console.log("");
+  console.log(Path.join(tmp, "peers"));
+  console.log(Path.join(tmp, "repos"));
```

Comment:
We should print the actual paths that are to be deleted here, so we should either store them in a variable and pass that to `removeWorkspace()` or combine the two functions. Otherwise if a dev changes the `removeWorkspace()` function and makes a mistake, the user will get the wrong idea of what is getting deleted.

Reply:
Done in 4c9466d